Kickstarter-iOS/Views/Controllers/CreatePasswordViewController.swift

```diff
@@ -64,27 +138,35 @@ final class CreatePasswordViewController: UITableViewController {
   }
 
   override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-    let row = CreatePasswordRow.allCases[indexPath.row]
     let cell = tableView.dequeueReusableCell(withClass: SettingsTextInputCell.self, for: indexPath)
 
     guard let textInputCell = cell as? SettingsTextInputCell else { return cell }
 
-    textInputCell.configure(with: row.title, placeholder: row.placeholder)
+    let row = CreatePasswordRow.allCases[indexPath.row]
+
+    textInputCell.configure(with: row.placeholder, returnKeyType: row.returnKeyType, title: row.title)
+    textInputCell.configure(with: textFieldTargetActions(for: self, row: row))
+
     return textInputCell
   }
 
   // MARK: - UITableViewDelegate
 
+  override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+    guard let textInputCell = tableView.cellForRow(at: indexPath) as? SettingsTextInputCell else { return }
+
+    textInputCell.textField.becomeFirstResponder()
+  }
+
   override func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
     let className = SettingsGroupedHeaderView.self
 
     guard let headerView = tableView.dequeueReusableHeaderFooterView(withClass: className)
       as? SettingsGroupedHeaderView else { return nil }
 
-    headerView.configure(
-      with: Strings.Well_ask_you_to_sign_back_into_the_Kickstarter_app_once_youve_changed_your_password()
-    )
+    let text = Strings.Well_ask_you_to_sign_back_into_the_Kickstarter_app_once_youve_changed_your_password()
 
+    headerView.label.text = text
```

Comment:
The same idea mentions on my previous comment would apply for the headerView as well.